js/mocha-tests/.editorconfig

```diff
@@ -0,0 +1,10 @@
+# http://editorconfig.org
```

Comment:
I'd leave `.editorconfig`out as I feel this is personal preference or at least something to be agreed upon.